### DISCUSSION TITLE
JLReq-d 編集会議 2024-10-24

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq-d#68] [General]
# JLReq-d 編集会議 2024-10-24

### 出席者
敏先生 @KobayashiToshi , 山口さん @yamahige , 木田 @kidayasuo 

### ノート
- 章の構成の変更（下のドラフト）：2章を「文字とフォント」だけにして、段落などは3章に。ここまでがユーザー向け。4章からが実装者向け。
- 前回の続き、2.2の中の漢字からレビュー。2.3の終わりまで。次は2.4から。
- 3章の「ルビ」の項に、ドラフト4.2のルビの使用例を含める。
- 2.3 の約物、木田がドラフトして次回レビュー。
- 次回10/31 10:00からオフライン。敏先生が会場を確保。

---- 
### 2　日本語デジタルテキスト入門　文字とフォント
現在の2章の文字とフォントの部分

### 3　日本語デジタルテキスト入門　文を作る
下記の内容を2章から分離する
- 組版の基本（べた、禁則、自動的に起こる）
- 段落と行そろえのスタイル
- ルビ
- 横書きと縦書き

### 4　日本語デジタルテキストの組版
現在の3章と4章

----

**Comment by kidayasuo** [comment]
アラビア数字と漢数字の説明、もう少しだけ詳しく、以下のように「詳細は省くが」と「専門用語」「など」を加えました。またコメントの中にあった英語の場合との対比をコメントから出しました。どうでしょう？

### アラビア数字と漢数字
歴史的には数字を漢字で示していたが、現代では数量を表す際にアラビア数字の使用が増えている。一方、詳細は省くが、固有名詞、慣用句や専門用語などの中に数字が現れる場合は漢字を使う。この使い分けは英語におけるアラビア数字とスペルアウトの用法に似ている。

（それぞれのコメントに対するコメントは、一番下の大きな"Add a comment"欄ではなく、各々のコメントの枠の下にくっついている"write a reply"で書くことができます）

**Comment by kidayasuo** [comment]
2.2 の約物の説明において、「句読点」の下に読点と句点、「その他の約物」の下に感嘆符と疑問符がありますが、感嘆符も疑問符も句点と同様に文を終える役物があり、これらが分かれているのは気持ちが悪いです。英語ではTerminal Punctuation（終止符）と総称されます。よって、これらはひとまとめにするのが良いのではいでしょうか？例えば：

#### 終止符
- 読点（、）
- 句点（。）
- 感嘆符（！）
- 疑問符（？）


ただ、読点は終止符ではありませんので、気持ち悪さが残ります。全体の分類を論理的に下のようにするのはどうでしょう？

#### 文の終わりを示す記号（終止符、Terminal Punctuation）
- 句点（。）
- 感嘆符（！）
- 疑問符（？）

#### 文の中の構造を示す記号（内部句読記号、Internal Punctuation）
- 読点（、）
- コロン
- セミコロン
- 中黒（・）
- ダッシュ（―, ー, 〜）
- 三点リーダ ……

#### ペアで使われる約物（Paired Punctuation）
- シングルクオーテーション（‘ ’）
- ダブルクオーテーション（“ ”）
- 丸括弧（（ ））
- 角括弧（［ ］）
- 鉤括弧（「 」）
- 二重鉤括弧（『 』）

**Comment by kidayasuo** [comment]
「約物には使い方の慣用があるのでそれに従う」の部分ですが、慣用、とは具体的に何でしょう。慣用がある、と書くと誰しも、どこに書いてある？　と思うでしょうから、例えば「記者ハンドブック」などと例を書いておきたいと思います。

**Comment by kidayasuo** [comment]
「2.3 文字の使い方」を下のように粗ドラフトしてみました。乞うご意見。

## 2.3 文字の使い方
デジタルテキストの特徴の一つは、印刷や手書きと異なり、文字を目に見えるイメージとして表すのではなく、文字を文字コードで表すことを基礎としている点にある。そのため、目の前のテキストが良ければそれで良いというわけではなく、文字コードが正しい必要がある。

たとえば、横書きの場合、長音とダーシは見た目がよく似ているが、誤って使っても読める一方で、意味が異なり、組版処理も違ってくる。また、誤った文字の使用は検索やその他のデジタル処理においても問題を引き起こし、意図した情報に正しくアクセスできなくなる可能性がある。

また、国際的な文字環境の下では、文字コードは正しいのに見えているものが正しくない、という現象も起こる。これはフォントが適切なものでない場合に起こり、漢字や約物の字形に影響が大きい。

よって、デジタルテキストにおいては正しい文字を使い、かつ正しいフォントを使用することが重要となる。

この節では、上記で述べたことを中心に、文字の使い方の注意点について述べる。

> ここでは文字の使い方、選び方を説明し、レイアウトの詳細には立ち入らない。

### 約物一般の使用方法
約物には使い方の慣用や規則があるのでそれに従って使用することが重要である。例えば会話や引用を鉤括弧で括るなど、広く一般的に受け入れられている慣用がある。また特定のコミュニティで規則が定められている場合もある。詳細については……

横書き、縦書きで使用する約物が異なる場合がある。これについては縦書きの節で解説する。

> 詳細には立ち入らないにしても、ここで、大まかな使用法は説明するか？　欧文約物に形と用法が似ている。つまり多くの言語で認知されている用法とほぼ同じ。句読点は英語のカンマとピリオド。?!は言わずもがな。括弧は英語とほぼ同じ、鉤括弧は英語のクオーテーションとほぼ同じ。コロンは？　セミコロンはほとんど使わない。範囲を示すのは英語ではハイフンだが、日本語では波ダッシュも使われる。ここは異なるので注意。

> ！＃＄％＆（）＊＋，－．／：；＜＝＞？［＼］＾＿｛｜｝～｟｠
> ＂(FULLWIDTH QUOTATION MARK)
> ＇(FULLWIDTH APOSTROPHE)
> ｀(FULLWIDTH GRAVE ACCENT)

> 特に括弧類に欧文約物にないものがある《》【	】〔〕〖〗〘〙〚〛〝〞〟
> が、それらが常に全幅とは限らないのに注意。

> ゲタ〓は文字のない場合に。特に漢字は数が多く、欲しい文字が手元にない場合に、ここに文字が欠けていること、やレイアウトを確認するために使われた。

EMダッシュと三点リーダは二つ続けて使用するのが基本。ダーシは長音と間違えないこと。また、罫線素片などダーシでないものを代替えとしない。

疑問符と感嘆符の後は一文字空ける。開け過ぎに感じる場合も多いが……。開けない場所も説明。

### 和文約物と欧文約物
先に述べたように和文約物は欧文に起源を持つものが多く、見かけの似ているものが多い。しかし同じように見えるものであっても、和文約物は和文文字に、欧文約物は欧文文字に合わせたデザインになっているので混用は望ましくない（望ましくない、よりも、具体的な表現が欲しい）。いくつかの代表的な例を挙げて説明する。

括弧には和文用の括弧と、ラテン文字用の括弧があるが、和文の括弧は和字を囲むようにデザインされているのに対し、ラテン文字用の括弧はラテン文字をディセンダを含めて囲むようにデザインされており、図のように上下位置が異なる。よって、和字にラテン文字用括弧を用いると、下にずれたような見かけになってしまう。この違いは、国際化環境などでラテン文字に使われるフォントと和字に使われるフォントが異なるときにさらに強調される。このような問題があるため、囲まれる文字に対応するものを使うのが基本である。
![](%E3%82%B9%E3%82%AF%E3%83%AA%E3%83%BC%E3%83%B3%E3%82%B7%E3%83%A7%E3%83%83%E3%83%88%202024-10-25%208.31.28.png)

括弧と同様に、ラテン文字用のクオーテーションマークは一般に和字用のものとはデザイン、特に上下位置や左右の空きが異なるので、囲まれる文字に対応するものを使うのが基本である。
![](%E3%82%B9%E3%82%AF%E3%83%AA%E3%83%BC%E3%83%B3%E3%82%B7%E3%83%A7%E3%83%83%E3%83%88%202024-10-24%2022.56.30.png)

囲まれる文字列の前端と後端の一方が和字でもう一方がラテン文字の場合、また「（きだ）（kida）」のように括弧やクオーテーションマークが連続したり、箇条書きなどで上下に並ぶような場合にはどちらかに統一する。

欧文とペアになる和文約物を列挙

### フォントの選択により切り替える約物
上記は対応する和文用と欧文用の約物が異なる文字コードを持つ例であるが、和文用と欧文用の約物が同じ文字コードを持つ例がある。

三点リーダとElipses。この二つの文字は同じ文字コードを持っており、フォントによって文字のデザインが異なる。

他にも、フォントの選択によりプロポーショナルと全幅が切り替わるものがある。

原則は、約物には日本語フォントを使う。

### 特殊な操作の必要な和文約物
クォーテーションマークのうち、左右で形の異なるより洗練されたデザインのものは、2024年時点で欧文用のみが存在し\*、これらの文字の和文用が存在しない。ただし、これらの文字コードに対して和文用の代替えグリフを持っているフォントもあり、フォントの高度な機能を使用できるアプリケーションにおいて使用することができる。
- LEFT DOUBLE QUOTATION MARK
- RIGHT DOUBLE QUOTATION MARK
- LEFT SINGLE QUOTATION MARK
- RIGHT SINGLE QUOTATION MARK

\* 正確には文字コードには欧文用との指定があるわけではなく、この文字コードに和文用のデザインを置くこともできるが、英語と日本語などその他の言語をサポートする必要のある環境では、この文字コードに欧文用のデザインが置かれている。

このような文字は、この他に、ダーシ？

### 全角英数字の使用
全角英数字には以下のような問題があるので使用には注意が必要である。

- 全角でないラテン文字や数字と混ぜて使うと視覚的一貫性を損なう。同じ文字でも、幅の異なるものが混在することになり、さらに国際化環境などでは英数字に和字とは異なるフォントが用いられる場合が多く、この場合には文字デザインの一貫性も損なわれる。
- 英字略語のように一文字一文字読む場合を除き、英単語や英文などに用いると可読性に劣る。また、行の折り返しが単語の途中で起きてしまう。
- 幅が通常の文字に比べて長いため、文字列が長くなる。
- アクセントの付いたものは存在せず、よってアクセント付き文字の必要な単語を表す時には使えない。
- 全角英数字はURLの構造的部分に使うべきではない。URLの構造的部分は通常の英数字を用いることが規定されており、全角英数字を使うと動作しないなどエラーの原因になり、読み手に負担をかけることになる。URL以外でも、HTML/CSSやmarkdownなどの機能的な部分に使用することはできない。

一方、全角英数字は縦書きで自動的に縦向きになるので、縦書きでの英字の記号的利用に便利である。

### 漢字
先に述べたように漢字は中国で生まれた文字である。しかし現在使われている漢字の形は、同じ文字コードであっても、日本、中国、台湾など、それぞれの漢字使用地域で相互に異なるので、フォントの使用に注意が必要である。多くの場合読めはするが、使用フォントを間違えると違和感のある、プロフェッショナルでない仕上がりとなる。

![](%E3%82%B9%E3%82%AF%E3%83%AA%E3%83%BC%E3%83%B3%E3%82%B7%E3%83%A7%E3%83%83%E3%83%88%202024-10-25%208.13.10.png)

**Reply by himorin** [reply]
既に後の版で変わっているかもですが、、

> デジタルテキストにおいては正しい文字を使い
ここは文字コードですかね。ただ、全般的にここでの「文字コード」はコードポイントに置き換えるべきな気はしました。

**Reply by kidayasuo** [reply]
good pointですね。ただ、ユーザー向けの最初の3章ではよりわかりやすくしたいので、文字、という言葉を用いて、わかりやすい場所に、少し詳しい説明を書くというアプローチの方が良いかもしれませんね。

**Comment by KobayashiToshi** [comment]
>「2.3 文字の使い方」を下のように粗ドラフトしてみました。乞うご意見。 2.3 文字の使い方

>約物には使い方の慣用や規則があるのでそれに従って使用することが重要である。例えば会話や引用を鉤括弧で括るなど、広く一般的に受け入れられている慣用がある。

ここでは，使い方一般は説明しないで，主に文字コードと関連して混用されがちなものに限定して説明したらどうかとも思うが．本格的に解説するか，入口でとどめるかも考えた方がよい．

もっとも用法で問題となる例に括弧があるが，これについては，“JDLReqメモ3_2.txt”の“2.3.9　括弧類の用法”に書いておいたが，これは，ここではいらないと思う．

テーマとしては，以下がある．

1　ラテン文字用と和文用
1.1　ラテン文字用と和文用　似たもののある例
1.2　和文用しかない例
1.3　ラテン文字にしかない例
＊これについては，“JDLReqメモ3_2.txt”の“2.3.10　日本語の約物と欧文の約物”に書いておいた．
1.4　コーテーションマークの問題

2　ダーシ類の扱い
2.1　ダーシと音引
2.2　語句を結び付ける方法と範囲の示し方
　語句を結び付ける方法　二重ダーシ，二分ダーシ，ハイフン
　範囲の示し方　二分ダーシ，全角ダーシ，波ダッシュ
＊漢数字の場合は，全角ダーシ，プロポーショナルなアラビア数字の場合は，二分ダーシだと思うが，最近は，漢数字の場合では波ダッシュ，プロポーショナルなアラビア数字の場合も波ダッシュが増えている．
2.3　全角ダーシはU+2014かU+2015か．

3　フォントにより字形が大きく異なる例
　3点リーダー

こう考えると，原則として和文では和文用を使うのが望ましいということと，長音には音引を使用しなしさいということ，コーテーションマークでは困っているというくらいでいいのではないかな．範囲を示す方法は，ふれなくてもよいとも思う．（全角ダーシはU+2014かU+2015の両方が使用されているというので注意が必要なくらいはいってもよいかもしれない）

それとも，詳細に説明するようにしますか？

**Reply by kidayasuo** [reply]
英語で、日本語における約物の使い方を説明しているものを見つけるのが難しいであろうことを考えると、少しだけ、説明しておく価値はあるかもしれませんね。とはいえ、jlreq-dは日本語の書き方を解説する文書ではありませんので、触れるにしても入り口程度で。かつ、括弧など他の言語から類推できるものは省いて違うところだけ書くのではどうでしょう？

その線で説明する場合、重要なのは：
- 「、。」がカンマ、ピリオドに対応する
- 「」がクオーテーションマークに対応すること
- EM DASH, 三点リーダは二つ続けて使う（JLReqでも解説）
- DASHと長音の混同に注意
- ？！の後は和字空白を入れる（JLReqでも解説）

程度か？

ところで、コメントに対してコメントする際には、各々のコメントの枠の一番下にある "Write a reply"のところに書くと、このように該当コメントとまとめて表示されてわかりやすくなります。

**Reply by kidayasuo** [reply]
U+2015 はUnicodeから言うと目的外使用ですので、あまりここでは触れたくないですね……

**Reply by KobayashiToshi** [reply]
基本的には木田さんの線でいいと思います．いくつか追加情報を
括弧の引用での使用方法，補足にブラケットを使うのは英文も同じようだ．キッコウは，ブラケットを日本風に変えたもので，ラテン文字には含まれない．
U+2015はやめましょう．
句読点は，3つの方式があることを書いてもよい
コーテーションマークは，
　「」　→　“”　『』→`’　が日本では多い．米国風かな．
三点リーダは1つで使う場合もある
 ？！の後は文末の場合は和字空白で空けるが，文中ではベタか，空ける場合でも狭くする．

**Comment by KobayashiToshi** [comment]
>「約物には使い方の慣用があるのでそれに従う」の部分ですが、慣用、とは具体的に何でしょう。慣用がある、と書くと誰しも、どこに書いてある？　と思うでしょうから、例えば「記者ハンドブック」などと例を書いておきたいと思います。

“記者ハンドブック”は，どちらかといえば，新聞と雑誌で参照される例が多い．朝日も読売も毎日，新聞協会その他にも，同様な本がある．

また，編集・校正関係の技術書（例：“標準編集必携”“標準校正必携”）でも，約物の使い方の記述はある．約物には使い方が最も問題となる原稿編集（原稿整理）については，日本では，その書名の本は以下しかない．
日本エディタースクール編“原稿編集ルールブック　第2版”日本エディタースクール出版部（2012.9）

最も詳細なのは，
小学館辞典編集部編“句読点、記号・符号活用辞典”（小学館，2007.9）

**Reply by kidayasuo** [reply]
「原稿編集ルールブック」が「慣用」を代表していると言えるなら、これを紹介しておきますか。

**Reply by KobayashiToshi** [reply]
“原稿編集ルールブック”では，約物の使い方は，1ページ少々しか書いてない．詳細な使い方を説明している以下がいいのではないかな．または両方．
小学館辞典編集部編“句読点、記号・符号活用辞典”（小学館，2007.9）

**Reply by KobayashiToshi** [reply]
“原稿編集ルールブック”や“句読点、記号・符号活用辞典”はやめて，必要なら，以下にしましょう．こうほうが簡潔で，網羅的．現在でも参考になります．
“くぎり符号の使ひ方”〔句読法〕（案）（文部省教科書局調査課国語調査室作成）
https://www.bunka.go.jp/kokugo_nihongo/sisaku/joho/joho/kijun/sanko/pdf/kugiri.pdf

**Reply by kidayasuo** [reply]
おお、まだ通用するのですね。これにしましょう。この文書の冒頭にあるように「参考にされている」というような紹介の仕方で。

**Reply by KobayashiToshi** [reply]
31日（木）に議論しましょう．

> 2.3 文字の使い方

1　タイトルは，以下はどうでしょう．使い方の全部を説明しているわけではないので．“上記で述べたことを中心に、文字の使い方の注意点について述べる”との説明もある．
　2.3　文字使用上の注意点
　2.3　文字の使い方の注意点

2　以下，デジタルテキストでも最終的には目に見えるイメージとするので，ちょっと気になる
> 文字を目に見えるイメージとして表すのではなく、

A案　この部分を削除
B案　最終的に文字を目に見えるイメージとして表す点では同じであるが，文字を文字コードとして準備することを基礎としている点に違いがある。

3　以下
> これはフォントが適切なものでない場合に起こり、漢字や約物の字形に影響が大きい。

以下はどうですか
これはフォントが適切でないと、漢字や約物に望ましくない字形の違いが出る場合がある。

4　以下
> また特定のコミュニティで規則が定められている場合もある。詳細については……

例えば，“くぎり符号の使ひ方”〔句読法〕（案）（文部省教科書局調査課国語調査室作成）*1は，今日でも参考にされている．

*1　https://www.bunka.go.jp/kokugo_nihongo/sisaku/joho/joho/kijun/sanko/pdf/kugiri.pdf
＊“新しい国語表記ハンドブック”（三省堂）では，この文書が掲載されています．

5　タイトルの“約物一般の使用方法”は，以下でどうですか
　約物の使い方

6　約物一般の使用方法の詳細
次のように簡単な説明でどうか
1）横書き、縦書きで使用する約物が異なる場合がある。これについては縦書きの節で解説する。
2）英文等とは異なる例をいくつか示しておく．
2.1）日本語では，欧文に起源を持つ約物が多く使用されており，その用法も似ている．しかし，これらの多くは和文用の文字が準備されている．詳細は事項で説明する．
2.2）横書きの句読点は，英文と同じコンマとピリオドを使用する方法がある．ただし，これも前項と同様に和文用の文字が準備されている．
2.3）括弧類では，英文等のフォントでは準備されていない以下のようなものがある．
　《》【 】〔〕〖〗〘〙〚〛〝〞〟
＊字幅は，ここではまだ触れなくてよいのでは？
〓はいうか？　これは約物か？
2.4）日本語用の3点リーダは，ボディの中央に字面を配置している．

EMダッシュと3点リーダは2つ続けて使用する，疑問符と感嘆符の後は1文字空けるは，組版処理なので，いわなくてもよいのでは？
ダーシは長音は，前にいっている．似た字形の約物をまとめてもよいが，…？　やめよう．
範囲を示す方法は，ラテン文字と同じ方法もあるし，波形を使う場合もあるし，漢数字でEMダッシュを使う方法もある．どれも間違いではない，けっこうややこしい．詳しく説明すると他とバランスを壊すかも．
23–35　23～35
二三—三五　二三～三五

7　和文約物と欧文約物　→　和文用約物と欧文用約物

＊ここは，修正したテキストを示す．

前述したように和文用約物には欧文に起源を持つものが多い。しかし，和文用約物と欧文用約物では，字形が似ているが，和文用約物は和文文字に、欧文用約物は欧文文字に合わせたデザインになっている．したがって，和文文脈では和文用約物を使用し，和文中に挿入される欧文には欧文用約物を使用するようにする．
＊同一ドキュメント内での混用はありえる
以下では，いくつか例を示しておく
1　コンマとピリオド　用法はほぼ同じであるが，和文用約物は必ずしもベースライン位置にあるとは限らない．したがって，アラビア数字の後ろに和文用約物を使用するとベースライン位置がそろわない場合がある．また，和文用は，その外側に余白を持っており，字幅も異なる．
2　丸括弧（パーレン）　ラテン文字用の括弧はラテン文字のディセンダを含めて囲むようにデザインされており、図のように上下位置が異なる。よって、和字にラテン文字用括弧を用いると、下にずれたような見かけになってしまう。また，和文用は，その外側に余白を持っており，字幅も異なる．この違いは、国際化環境などでラテン文字に使われるフォントと和字に使われるフォントが異なるときにさらに強調される。このような問題があるため、その文脈に対応するものを使うのが基本である。
ブラケットも同様である．なお，ブラケットに似せ，縦組用に変形した括弧にキッコウ（〔〕）があり，用法もブラケットに似たところがある．
＊“デイセンダ（descender）は，……”という場合，多くは，和文文脈と考え，括弧の中はラテン文字だが，パーレンは和文用を使うのがほとんどです．ですので，“囲まれる文字に対応する”とは必ずしもいえない．
3　疑問符（？）と感嘆符（！）　パーレンは和文用と欧文用では，一般に字幅が異なり，字形のデザインも異なる．特に縦書きでは，全幅の疑問符と感嘆符を使用している．横書きでも同様にであるが，全幅では幅が広すぎるとしてラテン文字用を使用する方法もあるが，これは例外的な使用方法と考えた方がよい．
4　アステリスク　記号ではあるが，和文用のアステリスクはボディの天地左右中央に字面が配置されているが，欧文用はボディの左右中央で上側に一般に配置されている．
5　ラテン文字用のクオーテーションマークは，その外側に余白を持っていないが，和字用は，余白を持っている．また，字形のデザインも異なる．

7　フォントの選択により切り替える約物

＊ここも修正したテキストを示す

上記は対応する和文用と欧文用の約物が異なる文字コードを持つ例であるが、和文用と欧文用の約物が同じ文字コードを持つ例がある。

3点リーダとElipses。この2つの文字は同じ文字コードを持っており、フォントによって文字のデザインが異なるだけでなく，和文用の字面はボディの天地中央であるが，欧文用はベースラインにそろえてある。

他にも、フォントの選択により，字幅がプロポーショナルと全幅が切り替わるものがある。

和文文脈の約物では，原則として日本語フォントを使う。

8　特殊な操作の必要な和文約物
特になし

9　全角英数字の使用

>　全角でないラテン文字や数字と混ぜて使うと視

同一ドキュメントで？　どの範囲でか？
同一テキスト内では　くらいか？

> 英単語や英文などに用いると可読性に劣る

英単語や英文などに【全角英数字を】用いると可読性に劣る

全角英数字では，以下のような問題もあります．

全角英数字は，いくらかボディの左右に余白がある文字が多い．したがって，これを横書きで使用すると，括弧類や句読点と全角英数字が連続すると，括弧類や句読点と全角英数字との字間に不必要な空白がでる場合がある．

その他　以下を入れるか？　こうすると〓も入れられる．
日本語のフォントには，①㋑ⓐのように文字を丸で囲った文字が含まれている．箇条書きの番号等に使用されている．また㈱㍻㍍のように1つのボディに複数の文字や約物をセットにしたものがある．これらは，普通の文章ではあまり使用されないが，スペースにない箇所等で使用する例がある．

**Comment by KobayashiToshi** [comment]
>歴史的には数字を漢字で示していたが、現代では数量を表す際にアラビア数字の使用が増えている。一方、詳細は省くが、固有名詞、慣用句や専門用語などの中に数字が現れる場合は漢字を使う。この使い分けは英語におけるアラビア数字とスペルアウトの用法に似ている。

固有名詞、慣用句などの中に数字が現れる場合は漢字を使う。専門用語でも、“二酸化炭素”のように、慣習として漢字を使う例がある。

専門用語で漢数字にするのは，ごく一部です．

**Reply by kidayasuo** [reply]
専門用語でアラビア数字を使う例にはどのようなものがありますか？

**Reply by KobayashiToshi** [reply]
今，思いつくのは化合物の名称だけかな．なぜか伝統的にそれが慣用のようです．“理科年表”には例が多数あります．
二酸化炭素
一酸化二水素
二酸化硫黄（SO2）
六価クロム

**Reply by kidayasuo** [reply]
どこだったかいま見つからないのですが、どこかに、「数が語の構成要素になっている場合」に漢数字を使う、と書いてありました。この表現はなかなか的確簡潔で良いかもしれません。専門用語、として「語」なら一般的には漢数字ではないでしょうか。「2次反応、1次試験、3次元」などの場合はアラビア数字ですが、「数が語の構成要素」とはなっていません。

**Reply by kidayasuo** [reply]
下はChatGTPさんに教えてもらった、英語で数字をスペルアウトするルール、および日本語の漢数字の使用との比較、です。

----
「数字が語の構成要素」となる場合に日本語で漢数字を使うのは、英語で数字をスペルアウト（文字で表記）するルールとよく似た感覚が見られます。英語でも日本語でも、こうした表記には形式的・伝統的な美しさや文脈の一貫性が重視されています。

### 英語におけるスペルアウトのルール

1. 一般的なルール
    10未満の数字は文字で表記することが推奨されます。
    - 例: “Three days passed.”（3日が経った）

    10以上は数字で書かれることが多いですが、正式な文書ではスペルアウトされることもあります。
    - 例: “Twenty students attended the lecture.”

2.	慣用句・固定表現
    慣用表現では、数字が文字として書かれることが一般的です。
    - 例: “At sixes and sevens”（混乱している）

3.	序数詞や特定の語句
    - 「Second World War」や「First Avenue」のように、正式名称や慣用表現ではスペルアウトされます。

### 日本語と英語の共通点

#### 慣用性の重視
日本語の「三つ子の魂百まで」や「一石二鳥」のような表現は、英語でも “Two heads are better than one” のように、数字が文字として表現されることで一体感が生まれます。

#### 正式性・格式の維持
英語ではフォーマルな文脈で数字をスペルアウトするように、日本語でも公式な文章や歴史的な名称では漢数字が使われます。
    - 例: 「第二次世界大戦」「四月一日」

### 違い

- 英語では10未満の数字をスペルアウトするルールがありますが、日本語ではそのような数値基準は存在せず、文脈や慣例に依存します。
- 日本語の漢数字は、美的・伝統的な理由からも使われやすい一方で、英語では実用性に基づいてスペルアウトと数字表記を使い分けます。

### まとめ

日本語の漢数字の使用と英語のスペルアウトには、形式や一貫性を重視する共通点があります。ただし、英語は数字の大きさに基づいたルールが明確である一方、日本語では慣習や文脈がより強く表記を決定します。このような使い分けが、双方の言語の表現に独特のリズムと美しさを与えています。

**Reply by KobayashiToshi** [reply]
うーん，英語のスペルアウトと日本語の漢数字の使い方は，似ているようで，違うのではないかな．英語のスペルアウトを日本語でやれば，ひとつ，ふたつ，みっつ，…，でしょう．数を表す文字が違うというのが，漢数字か，アラビア数字かという違い．ローマ数字を使う例もある．

また，序数詞は，漢数字にする例もあったが，最近はアラビア数字が多い．例：第ニ次世界大戦，第2次世界大戦
訓読みの数字の“二つ”も最近は，“2つ”が，これがけっこう多くなっている．
年月日は，最近は縦組でもアラビア数字が多い．東京新聞は，漢数字でがんばっていたが，今年かな，アラビア数字に転向した．
五・一五事件，二・二六事件とも書くが，縦組でも5・15事件，2・26事件とアラビア数字にする例が多いでしょう．

> 専門用語、として「語」なら一般的には漢数字ではないでしょうか。

今は使われていない除草剤のジクロルフェノキシ酢酸は“2,4–D”は，言葉だが，アラビア数字を使用している．“2,4”は，基の位置を示す数字だからということのようです．倍数接頭辞だけが漢数字を使う．

**Reply by kidayasuo** [reply]
英語のスペルアウトと日本語の漢数字の共通点は、両方とも、公式な文書において長らく使われてきた表現であるということです。これに対して、数量を表すのに優れたアラビア数字が、英語においても、日本語においても入ってきた、という歴史の共通点があるように思います。

2,4-Dは化学物質の構造を表す数式的な表現であって、「数字が語の構成要素」の場合とは異なるように思います。もちろん、商品名などでは自由な表現で作れますので、いくらでも例外は考えられますが。

目的は、こうしなさい、という指示ではなくて、一般的な用法の傾向を示したいだけですので、簡潔で良いかと思います。

**Comment by KobayashiToshi** [comment]
>例えば「記者ハンドブック」などと例を書いておきたいと思います。

手元にある本が見つからないので，記憶ですが，この本では，約物の使い方の記述は，あまりなかったんではないかな．

**Comment by kidayasuo** [comment]
ところで、全角ダッシュや三点リーダを、二つ繰り返さずに、単独で使うケースはあるのですか？

**Reply by KobayashiToshi** [reply]
全角ダッシュは，一九五六—一九七〇年のように漢数字で範囲を示す場合に使う．また，括弧と同様な使用法は2倍が多いが，全角もたまに見る．書名（見出し）に対するサブタイトルとの区切りも2倍が多いが，全角も，たまに見る．なお，範囲の示し方は，かつては全角ダーシが一般的であったが，最近は波形が多い．もちろん全角ダーシの使用例もあるが，少数派．（書名に対するサブタイトルとの区切りも2倍は長すぎるように思う．個別処理で1.5倍なんかにすることもある．）
三点リーダは，1, 23，…，nといった例がある．省略を示す場合は，多くが2倍だが1倍という例もある．“…〔略〕…”みたいな書き方もある．なお，2倍3点リーダーは，分割禁止だが，許容の例も，けっこう見掛ける．

感想：これまでのやり取りを見ると，木田さんは，どっちかというと漢数字派のようで，それに対し，私はアラビア数字派のようですね．

**Reply by kidayasuo** [reply]
単独でも使うんですね。必ず二つ続けるのなら、なぜその半分が文字になっているのかなという疑問が出てきますが、これでスッキリしました。では必ず二つ続けて使う、と書いてはダメですね。

>感想：これまでのやり取りを見ると，木田さんは，どっちかというと漢数字派のようで，それに対し，私はアラビア数字派のようですね．

意識的にそういうわけではないのですが、「第二次世界大戦」「一つ、二つ」の方がしっくりくる、という意味ではそうかもしれませんね。しかし年号ははっきりアラビア数字派です; 1956-1970。
